src/DynamoCoreWpf/ViewModels/Watch3D/HelixWatch3DViewModel.cs

```diff
@@ -54,24 +54,45 @@ public class CameraData
         private readonly Vector3D defaultCameraUpDirection = new Vector3D(0, 1, 0);
         private const double defaultNearPlaneDistance = 0.1;
         private const double defaultFarPlaneDistance = 10000000;
-         
-        public Point3D EyePosition { get; set; }
-        public Vector3D UpDirection { get; set; }
-        public Vector3D LookDirection { get; set; }
-        public string Name { get; set; }
+
+        [JsonIgnore]
+        public Point3D EyePosition { get { return new Point3D(EyeX, EyeY, EyeZ); } }
+        [JsonIgnore]
+        public Vector3D UpDirection { get { return new Vector3D(UpX, UpY, UpZ); } }
+        [JsonIgnore]
+        public Vector3D LookDirection { get { return new Vector3D(LookX, LookY, LookZ); } }
         [JsonIgnore]
         public double NearPlaneDistance { get; set; }
         [JsonIgnore]
         public double FarPlaneDistance { get; set; }
 
+        // JSON camera data
+        public string Name { get; set; }
+        public double EyeX { get; set; }
+        public double EyeY { get; set; }
+        public double EyeZ { get; set; }
+        public double LookX { get; set; }
+        public double LookY { get; set; }
+        public double LookZ { get; set; }
+        public double UpX { get; set; }
+        public double UpY { get; set; }
+        public double UpZ { get; set; }
```

Comment:
@QilongTang Added to the description.  I am also now returning numbers instead of strings to match the CoGS format